src/Simple.OData.Client.Core/Fluent/FluentClientBase.cs

```diff
@@ -14,7 +13,7 @@ namespace Simple.OData.Client
     /// </summary>
     /// <typeparam name="T">The entity type.</typeparam>
     /// <typeparam name="FT"></typeparam>
-    public abstract partial class FluentClientBase<T, FT> : IFluentClient<T, FT>
+    public abstract class FluentClientBase<T, FT> : IFluentClient<T, FT>
```

Comment:
Why was `partial` dropped?

Reply:
The implementation of FluentClientBase contains only this single part so `partial` is not necessary here. This change is not related to my other changes so I can revert this.